runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -399,6 +401,20 @@ public void init(@Nullable final Applet client) throws Exception
 		});
 	}
 
+	private boolean showWarningonExit()
+	{
+		if(config.warningOnExit() == WarningOnExit.ALWAYS)
+		{
+			return true;
+		}
+		else if(config.warningOnExit() == WarningOnExit.ONLY_WHEN_LOGGED_IN)
```

Comment:
No need for else if here as you are returning early in first if. also, same issue, space between if and (